amp-pwa-reader/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ShadowReader",
-  "version": "0.0.1",
+  "version": "0.0.2",
```

Comment:
I bumped up the version number.   Should we bump this up even more ambitiously?

Reply:
Let's just bump to 1.0.

Reply:
Yay.  We should get a cake!

Reply:
indeed :D